feedbacks.js

```diff
@@ -0,0 +1,69 @@
+var choices = require('./choices');
+exports.getFeedbacks = function (instance) {
+	var self = instance;
+	return {
+		feedback_loaded: {
```

Comment:
In general I'd like to see these feedback keys have the `feedback_` portion removed since its redundant

Reply:
Ok, I will remove the "feedback_" portion in the key.